### PR TITLE
New Biosuit Sprites by EvenInDeathIStillServe

### PR DESCRIPTION
**Pull request by ghost**
:cl: EvenInDeathIStillServe
add: Biosuit sprites are new!
/:cl:

Spritebot, play me off!

### CODE REVIEW

**Comment by cyclowns** [comment]
woah these look dope as hell

**Comment by optimumtact** [comment]
in game picture

**Comment by Armhulen** [comment]
Imho the masks need more of that filter bit. I'm gonna touch up the head sprites and see what you folk think

**Comment by ShizCalev** [comment]
Would you be so kind to do inhands for them too please?

**Comment by JJRcop** [comment]
Could you get EvenInDeathIStillServe to make a new helmet sprite inspired by the old bio_scientist helmet? I really like the look of it and want to see what he can do

**Comment by Armhulen** [comment]
@ShizCalev i'll see what I can do

File: code/modules/cargo/exports/gear.dm [hunk 1]
```diff
@@ -126,4 +126,4 @@
 /datum/export/gear/bombsuit
 	cost = 300
 	unit_name = "bomb suit"
-	export_types = list(/obj/item/clothing/suit/bomb_suit)
\ No newline at end of file
+	export_types = list(/obj/item/clothing/suit/bomb_suit)
```
Comment (nichlas0010) [inline code/modules/cargo/exports/gear.dm]: 🤔

Comment (Jalleo) [reply code/modules/cargo/exports/gear.dm]: It can happen. Line endings and such

**Review by WJohn — CHANGES_REQUESTED** [review]
Having seen these on discord I am not a fan of the square helmet. The red horizontal stripe on the sec suit is also gaudy as hell, as is the blue biohazard sign the size of the entire torso on the med suit.

Big 👎 

Here's a pic

**Comment by Armhulen** [comment]
Imo easy fixes. Add a little to the top so the perspective fits, more filter when facing sideways, and readd the vest on the sec biosuit.

**Comment by nicbn** [comment]
These look pretty bad

**Comment by Iamgoofball** [comment]
yeah wtf wj, what are you doing man

**Comment by Iamgoofball** [comment]
yeah if you check the actual spritebot it shows those sprites as much crisper and much better, something is wrong here

**Comment by ExcessiveUseOfCobblestone** [comment]
3/4th walls look like a mistake.

**Comment by Barhandar** [comment]
we exadvbiosuit now

And I mean these square helmets fit more on the original codersprite spessmen than current ones.

**Comment by JJRcop** [comment]
Take your own screenshot in-game, WJ based his verdict on a screenshot from discord

**Pull request closed without merging**